packages/common/src/hooks/useDirtyCheck/useDirtyCheck.ts

```diff
@@ -1,16 +1,11 @@
-import { useRef } from 'react';
+import create from 'zustand';
```

Comment:
aaaand, this didn't work. The hook rerendered and created a new ref, meaning the form is never dirty and new lines are never saved.

Could probably fix that.. but I went with option (a) and just reverted.

Reply:
I thought we wanted this hook to avoid using global state -- as per last week's discussion? So when you say "reverted" here, you mean put it back to how it was before my changes? 😄

Reply:
yep. exactly - just put the global state back in 😜  
the ref version wasn't working for me.. when editing the `dirty` flag kept changing `false` 🤷